client_plugin/drivers/shared/dockerops/dockerops.go

```diff
@@ -0,0 +1,82 @@
+// Copyright 2017 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package dockerops
+
+import (
+	"context"
+	log "github.com/Sirupsen/logrus"
+	dockerClient "github.com/docker/engine-api/client"
+	dockerTypes "github.com/docker/engine-api/types"
+)
+
+const (
+	// dockerAPIVersion: docker engine 1.24 and above support this api version
+	dockerAPIVersion = "v1.24"
+	// dockerUSocket: Unix socket on which Docker engine is listening
+	dockerUSocket = "unix:///var/run/docker.sock"
+)
+
+// DockerOps is the interface for docker host related operations
+type DockerOps struct {
```

Comment:
It's not only for create and manage volumes... We have all the operations who will directly talk to docker host sitting in this struct. We will also need to do operations like launch services in this package too.